pages.pl/common/tig.md

```diff
@@ -0,0 +1,28 @@
+# tig
+
+> Interfejs tekstowy dla Gita.
+> Więcej informacji: <https://github.com/jonas/tig>.
+
+- Pokaż listę commitów w odwrotnej kolejności chronologicznej, zaczynając od najnowszego:
+
+`tig`
+
+- Pokaż historię wybranej gałęzi:
+
+`tig {{gałąź}}`
+
+- Pokaż historię wybranych plików lub katalogów:
+
+`tig {{ścieżka1 ścieżka2 …}}`
```

Comment:
Oh, no. This char is not supported in some terminal configurations
```suggestion
`tig {{ścieżka1 ścieżka2 ...}}`
```
Please change the same in the English page.

Reply:
TIL! Many thanks for the comment, fixed in both files.